framework/src/main/java/org/checkerframework/framework/type/QualifierHierarchy.java

```diff
@@ -32,46 +41,56 @@
     /**
      * Returns the width of this hierarchy, i.e. the expected number of annotations on any valid
      * type.
+     *
+     * @return the width of this QualifierHierarchy
      */
-    public int getWidth() {
+    default int getWidth() {
         return getTopAnnotations().size();
     }
 
     /**
-     * Returns the top (ultimate super) type qualifiers in the type system.
+     * Returns the top (ultimate super) type qualifiers in the type system. The size of this set is
+     * equal to {@link #getWidth}.
      *
      * @return the top (ultimate super) type qualifiers in the type system
      */
-    public abstract Set<? extends AnnotationMirror> getTopAnnotations();
+    Set<? extends AnnotationMirror> getTopAnnotations();
 
     /**
      * Return the top qualifier for the given qualifier, that is, the qualifier that is a supertype
-     * of start but no further supertypes exist.
+     * of {@code qualifier} but no further supertypes exist.
+     *
+     * @param qualifier any qualifier from one of the type hierarchies represented by this
```

Comment:
Here and in a few other comments, should the "type hierarchies" be replaced with "qualifier hierarchies", to not confuse with TypeHierarchy instances? Sometimes it's the combined "type qualifier hierarchies".